docs/en/week08/08-3.md

```diff
@@ -0,0 +1,266 @@
+
+<center>
+<img src="{{site.baseurl}}/images/week08/08-3/bubbles_z.png" style="background-color:#DCDCDC;"  /><br>
+<b>Fig. 8</b>: Visualizing vector z as bubbles in the latent space
+</center>
+
+*In Fig. 8 above, each bubble represents an estimated region of $\boldsymbol{z}$, and the arrows represent how the reconstruction term pushes each estimated value away from the others, which is explained more below.*
+
+As stated above, the Free Energy for the VAE contains two terms: a reconstruction term and a regularization term.  We can write this as
+
+$$
+\tilde{F}(\boldsymbol{y}) = C(\boldsymbol{y}, \tilde{\boldsymbol{y}}) + \beta D_{\text{KL}}(\mathcal{N}(\boldsymbol{\mu}, \boldsymbol{v}) \mathrel{\Vert} \mathcal{N}(\boldsymbol{0}, \boldsymbol{1}))
+$$
+
+To visualize the purpose of each term in the free energy, we can think of each estimated $\boldsymbol{z}$ value as a circle in $2d$ space, where the centre of the circle is $\boldsymbol{\mu}$ and the surrounding area are the possible values of $\boldsymbol{z}$ determined by $\boldsymbol{v}$.
+
+If there is an overlap between any two estimates of $z$, (visually, if two bubbles overlap) this creates ambiguity for reconstruction because the points in the overlap can be mapped to both original inputs. The first term $C(\boldsymbol{y}, \tilde{\boldsymbol{y}})$ forces to reconstructing these bubbles to the correct locations. 
+- The reconstruction loss $C(\boldsymbol{y}, \tilde{\boldsymbol{y}})$ will push the points away from one another such there is no overlap.
+- Another option for $C(\boldsymbol{y}, \tilde{\boldsymbol{y}})$ to avoid overlap is to make the variance zero and then there are just points other than bubbles so there's no more overlap. 
+
+However, if we use just the reconstruction loss, the estimates will continue to be pushed away from each other and the system could blow up.  This is where the penalty term comes in.
+
+**The penalty term**
+
+The second term is the relative entropy (a measure of the distance between two distributions) between a Gaussian with mean $\boldsymbol{\mu}$, variance $\boldsymbol{\mu}$ and the standard normal distribution. If we expand this second term in the VAE loss function we get:
+
+$$
+\beta D_{\text{KL}}(\mathcal{N}(\boldsymbol{\mu}, \boldsymbol{v}) \mathrel{\Vert} \mathcal{N}(\boldsymbol{0}, \boldsymbol{1})) = \frac{\beta}{2} \sum\limits_{i=1}^d(v_i - \log{(v_i)} - 1 + \mu_i^2)
+$$
+
+where the expression in the summation has four terms. Let's use $V$ for the first three terms.
+
+$$
+V = v_i - \log{(v_i)} - 1
+$$
+
+From the left bottom plot in Fig. 8 above, we can see that this expression is minimized when $v_i$ is 1. Therefore our penalty loss will keep the variance of our estimated latent variables at around 1.  Visually, this means our "bubbles" from above will have a radius of around 1.
+
+The last term, $\mu_i^2$, minimizes the distance between the bubbles and therefore prevents the "exploding" encouraged by the reconstruction term.
```

Comment:
You haven't described `U` shown in figures 5 and 6…

Reply:
I mensioned "And similar to using regularisation factor R for h, we now use regularisation factors U and V for μ and v, respectively." under fig.5.